### PR TITLE
chore(tables): update prop descriptions

### PR DESCRIPTION
**Pull request by Mikaelia**
## Description

Updates tables package prop descriptions

### CODE REVIEW

File: packages/tables/src/styled/StyledCell.ts [hunk 1]
```diff
@@ -19,14 +19,13 @@ export interface IStyledCellProps {
...
   isTruncated?: boolean;
-  /**
-   * Styling for a cell that contains an `OverflowButton`
-   */
+  /** Applies styling for a cell that contains an overflow menu */
   hasOverflow?: boolean;
-  /** The width of the cell */
+  /** Adjusts the width of the cell */
   width?: string | number;
+  /** Adjusts the vertical pading of the cell */
```
Comment (jzempel) [inline packages/tables/src/styled/StyledCell.ts:28]: *padding

**Review by jzempel — CHANGES_REQUESTED** [review]
It seems like some of the elements listed here got missed:

https://garden.zendesk.com/components/table#api

Please double-check against `yarn start:styleguidist` to ensure all props are getting coverage.

File: packages/tables/src/styled/StyledCell.ts (hunk 1 above)
Comment (Mikaelia) [reply packages/tables/src/styled/StyledCell.ts:28]: Hmm I'm not sure why they are missing. I have run yarn start:styleguidist but it actually doesn't look like it is picking up any of my props in the `styled` folder..

Comment (jzempel) [reply packages/tables/src/styled/StyledCell.ts:28]: Yep, I just double-checked and the rendered docs are looking good, and only https://github.com/zendeskgarden/react-components/blob/67b39a7787e3f51560a8f60d84a0d6eb21192e4b/packages/tables/src/elements/SortableCell.tsx#L20-L22 is missing.

Comment (Mikaelia) [reply packages/tables/src/styled/StyledCell.ts]: Forgot to push 😅

**Review by jzempel — APPROVED** [review]

**Pull request merged**